src/index.test.ts

```diff
@@ -1,116 +1,208 @@
-import 'jest';
-import { isPresent, isDefined, isFilled } from '.';
+import "jest";
+import {
+  isPresent,
+  isDefined,
+  isFilled,
+  hasPresentKey,
+  hasValueAtKey,
+} from ".";
 
 type TestData = {
   data: string;
 };
 
-describe('Functions', () => {
-  describe('isPresent', () => {
-    it('should return true on present', () => {
-      expect(isPresent<TestData>({ data: 'hello world' })).toBeTruthy();
+describe("Functions", () => {
+  describe("isPresent", () => {
+    it("should return true on present", () => {
+      expect(
+        isPresent<TestData>({ data: "hello world" })
+      ).toBeTruthy();
```

Comment:
This formatting change is unexpected and I'll be reverting it and adding in tslint as a fast-follow. :)

Reply:
Decided to show those changes here: https://github.com/robertmassaioli/ts-is-present/pull/2

Please feel free to pull these changes in, or I can just merge that PR directly? Your call. 😄